webpack.config.js

```diff
@@ -62,7 +62,7 @@ module.exports = (env, argv) => {
           ]
         },
         {
-          test: /\.(png|woff|woff2|eot|ttf)$/,
+          test: /\.(png|woff|woff2|asc|eot|ttf)$/,
```

Comment:
You should only need to list `asc` once in this file, and I think it should be the `asset/resource` section below, so you should be able to remove it from here.